packages/office-ui-fabric-react/src/components/DatePicker/examples/DatePicker.Basic.Example.tsx

```diff
@@ -24,6 +24,25 @@ export interface IDatePickerBasicExampleState {
   firstDayOfWeek?: DayOfWeek;
 }
 
+interface IDatePickerExampleObject {
+  datePickerExample: string;
+}
+
+const classNames: IDatePickerExampleObject = mergeStyleSets({
+  datePickerExample: {
+    selectors: {
+      '&.ms-DatePicker': {
```

Comment:
Targeting the class names isn't currently the recommended approach. Instead you can define a shared object up here for the styles (since they're the same for both components) then pass that in via the `styles` prop to each component.

```
// here (choose some other name if you prefer)
const sharedStyles: IStyle = {
  margin: '0 0 15px 0',
  maxWidth: 300
};

// later
<DatePicker styles={{ root: sharedStyles }} .../>
<Dropdown styles={{ dropdown: sharedStyles }} ... />
```

Also, if you did want to target children using class names, you'd need to get rid of the `&` at the beginning of each selector--the current definition would only target `.datePickerExample.ms-DatePicker` or `.datePickerExample.ms-Dropdown`, which I don't think is what you intended.

Reply:
Same comment applies to the other files.

Reply:
Some guidelines on how to distinguish when to use the `styles` prop vs. `mergeStyles` or `mergeStyleSets`:
- To style a Fabric component, use the `styles` prop (usually)
- To style native elements like `div`, `span`, etc:
  - If only 1-2 class names are needed, use `mergeStyles` (returns a single class name)
  - If more than that, use `mergeStyleSets`